website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -278,4 +278,12 @@ HashiCorp collects the following utilization data as JSON payloads:
 }
 ```
 
-</CodeBlockConfig>
\ No newline at end of file
+</CodeBlockConfig>
+
+## Pre-1.9 Counts
```

Comment:
I could be mistaken (haven't looked at the deployed docs) but I think you need whitespace between headings and text.